runtime/calamari/src/lib.rs

```diff
@@ -792,6 +793,8 @@ parameter_types! {
 
 parameter_types! {
 	pub const ExecutiveBody: BodyId = BodyId::Executive;
+	pub const PerformancePercentileToConsiderForKick : Percent = Percent::from_percent(80);
+	pub const UnderperformPercentileByPercentToKick  : Percent = Percent::from_percent(10);
```

Comment:
Can we rename `UnderperformPercentileByPercentToKick` to something without `percentile` in the name.
`UnderperformancePrecentToKick` maybe.
Maybe it's just me but mixing percent and percentile got me confused a bit.

Reply:
I've been unhappy with the wording too.
I'm currently leaning towards calling it `EvictionPercentile` and `EvictionThreshold`

Reply:
Modified as above in the latest commit